pages/common/ftp.md

```diff
@@ -0,0 +1,27 @@
+# ftp
+
+> Internet File Transfer Protocol
+
+- Connect to a FTP server:
+
+`ftp {{ftp.example.com}}`
+
+- Switch to binary transfer mode (graphics, compressed files etc):
+
+`binary`
+
+- Turn off interactive prompting for multiple file transfers:
```

Comment:
Any suggestions for how to write this in a succinct manner? Here's the full `man` entry for `prompt`:

> `prompt`
>
> Toggle interactive prompting.  Interactive prompting occurs during multiple file transfers to allow the user to selectively retrieve or store files.  If prompting is turned off (default is on), any `mget` or `mput` will transfer all files, and any `mdelete` will delete all files.
> 
> When prompting is on, the following commands are available at a prompt:
> * `a`   Answer `yes` to the current file, and automatically answer `yes` to any remaining files for the current command.
> * `n`  Answer `no`, and do not transfer the file.
> * `p`  Answer `yes` to the current file, and turn off prompt mode (as is `prompt off` had been given).
> * `q`  Terminate the current operation.
> * `y`   Answer `yes`, and transfer the file.
> * `?`   Display a help message.
>
> Any other response will answer `yes` to the current file.

Perhaps something like _Transfer multiple files without interactive confirmations:_ or something?